src/app/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Home from '../components/home/home';
+import Info from '../components/info/info';
+import Nav from '../components/nav/Nav';
+
+function App() {
+  return (
+    <Router>
+      <Nav />
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/Info/:countryId" element={<Info />} />
```

Comment:
- [ ] Remove this element from navbar since it leads to an empty page 
![image](https://user-images.githubusercontent.com/62705947/170517715-3eee3cc8-39ac-4f8f-8dc3-f7e83b64b8cb.png)